Split 'controls and interface' tab into two tabs

AaronVanGeffen opened the pull request:
In the current Options window, it's unintuitive which of the tabs contains the control settings. I believe breaking up 'controls and interface' into two will make this much more intuitive.

After:
![Diamond Heights 2025-04-18 16-41-20](https://github.com/user-attachments/assets/8e3554f0-00a9-49aa-a3c0-be7a18d69e11) ![Diamond Heights 2025-04-18 16-41-21](https://github.com/user-attachments/assets/3615297a-aad8-4fe0-9b1c-90ada5833dd3)

Before:
![Grand Glacier 2025-04-18 16-41-27](https://github.com/user-attachments/assets/41d7cb0a-86fe-49fd-a3a4-5bc3e4cc1ccd)

J-PIE-314 commented:
Is there any way the miscellaneous tab could be done-away (options moved to other tabs)? Maybe that would be for another PR?

For example: Move Title Sequence to the new Interface tab.

AaronVanGeffen commented:
Yes, I don't like that we have both a 'Miscellaneous' and 'Advanced' tab either. They should be reworked, but we're deliberately keeping the scope smaller for this PR.

AaronVanGeffen commented on src/openrct2-ui/UiStringIds.h:
```diff
@@ -1152,12 +1152,13 @@ namespace OpenRCT2
         STR_OPTIONS_AUDIO_TIP = 5268,
         STR_OPTIONS_AUTOSAVE_FREQUENCY_LABEL = 2700,
-        STR_OPTIONS_CONTROLS_AND_INTERFACE_TIP = 5269,
+        STR_OPTIONS_CONTROLS_TIP = 5269,
```
To do: looks like the actual modified 'controls' string got lost along the way. I'll have to re-modify it.

LordMarcel commented:
The paint icon in the options animates too fast in this PR.

https://github.com/user-attachments/assets/a273f4c6-12cb-4492-8806-da5897affaae

AaronVanGeffen commented:
@LordMarcel Thanks for testing. It was using the wrong divisor. Should be fixed now!

Gymnasiast reviewed (changes requested):
The paintbrush animation is still too fast, despite Marcel’s comment being marked as solved.

AaronVanGeffen commented:
> The paintbrush animation is still too fast, despite Marcel’s comment being marked as solved.

You're right; the commit that fixed it was lost in a rebase. Cherry-picked it as 9689026.

AaronVanGeffen commented:
Rebased to account for recent changelog additions, and squashed a few commits. (Yes, I made sure to check the tab animation divisors were still correct! 😉)
